### PR TITLE
PMM release 2.41.0

### PR DESCRIPTION
**Pull request by rasika-chivate**
Add all the doc taks for PMM 2.41.0 in this branch only.

### CODE REVIEW

File: docs/get-started/query-analytics.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ The dashboard contains three panels:
 !!! note alert alert-primary ""
     Query Analytics data retrieval is not instantaneous and can be delayed due to network conditions. In such situations *no data* is reported and a gap appears in the sparkline.
 
+    During a network outage, QAN data continues to be gathered and is delivered as soon as the connection is re-established. 
+
```
Comment (BupycHuk) [inline docs/get-started/query-analytics.md:24]:
```suggestion
    Query Analytics data retrieval is not instantaneous and can be delayed due to network conditions. In such situations QAN data continues to be gathered and is delivered as soon as the connection is re-established. .

```

**Review by sleto-it — CHANGES_REQUESTED** [review]

File: docs/release-notes/2.41.0.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+
+--->
+
+## Release Highlights
+
+### Continuous data collection
```
[the remark this answers is not in the document]
Comment (rasika-chivate) [reply docs/release-notes/2.41.0.md:23]: @BupycHuk please provide your input on this.

[the remark this answers is not in the document]
Comment (sleto-it) [reply docs/release-notes/2.41.0.md:23]: mentioning a jira ticket can also be useful to understand more.  @rasika-chivate to me it makes also sense to link the new page how-to/HA from 2.41.md - so people can learn more.  Same when we talk in the next section about PMM dump makes sense to link the new page how-to/PMM_dump.md - thanks

[the remark this answers is not in the document]
Comment (BupycHuk) [reply docs/release-notes/2.41.0.md:23]: it's extension of that. In 2.33 we introduced offline metric collection for prometheus exporters, now we extended it by QAN metrics.

[the remark this answers is not in the document]
Comment (sleto-it) [reply docs/release-notes/2.41.0.md:23]: okay - thanks @BupycHuk this helps.  based on what you said, i think we can describe better this point. I will suggest something later..

[the remark this answers is not in the document]
Comment (rasika-chivate) [reply docs/release-notes/2.41.0.md:23]: > mentioning a jira ticket can also be useful to understand more. @rasika-chivate to me it makes also sense to link the new page how-to/HA from 2.41.md - so people can learn more. Same when we talk in the next section about PMM dump makes sense to link the new page how-to/PMM_dump.md - thanks

I will add

[the remark this answers is not in the document]
Comment (sleto-it) [reply docs/release-notes/2.41.0.md:23]: After the "Continuous data collection" section, and before the "Streamlined database problem reporting to Percona" we can add new section, something like:

```
Technical Preview of PMM in HA mode

Starting with v2.41.0 it is possible to start PMM in HA (high-availability) mode where three PMM Server instances are configured, one being the leader and others are followers.  
This feature is currently in Technical Preview. Early adopters are advised to use this feature for testing purposes only as it is subject to change. --> link to the HA page
```

[the remark this answers is not in the document]
Comment (sleto-it) [reply docs/release-notes/2.41.0.md]: Thanks for addressing this.  Resolving.  Thank you.

**Review by sleto-it — APPROVED** [review]
I did not review the entire PR, but only some parts.  On the parts I reviewed, the feedback that I shared has been implemented.  I am therefore approving.  Thank you.

File: docs/release-notes/2.41.0.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+### Offline metric collection
+
+The offline metric collection introduced in v2.33.0 has now been extended to QAN metrics. Previously, it was limited to Prometheus exporters only.
+
+PMM data is crucial for monitoring and troubleshooting. PMM operates on a client-server architecture, exchanging data between clients (on database servers) and servers (PMM UI interface and data storage). 
+
+Previously, if the PMM client-server connection was interrupted by a network or application issue, the system would stop collecting data, resulting in data collection interruption.
+
+Starting with PMM 2.41.0, if the PMM client and server components encounter connection problems, the QAN data gathered during these periods is stored and transmitted immediately after the connection is restored.
```
Comment (BupycHuk) [inline docs/release-notes/2.41.0.md:31]: @rasika-chivate @catalinaadam we made a decission to revert this changes, so let's remove it from the RN and docs

File: docs/release-notes/2.41.0.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rasika-chivate) [reply docs/release-notes/2.41.0.md]: @BupycHuk This will be out of scope for PMM 2.41.0 right?

File: docs/release-notes/2.41.0.md (hunk 3 above)
Comment (rasika-chivate) [reply docs/release-notes/2.41.0.md:31]: Yes. Will revert it.

File: mkdocs-base.yml [hunk 4]
```diff
@@ -178,17 +178,20 @@ nav:
           - get-started/roles-and-permissions/use_case.md
       - get-started/query-analytics.md
       - get-started/advisors.md
+      - get-started/HA.md
```
Comment (sleto-it) [inline mkdocs-base.yml:181]: @rasika-chivate this entry seems old, as we then moved the HA page under how-to.  Should we remove this line? see how it looks like in the preview, FYI (see that "none" after "working with advisors")

none

Comment (rasika-chivate) [reply mkdocs-base.yml]: Done

**Review by BupycHuk — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]
